packages/create-app/templates/default-app/package.json.hbs

```diff
@@ -30,7 +30,7 @@
   "devDependencies": {
     "@backstage/cli": "^{{version '@backstage/cli'}}",
     "@spotify/prettier-config": "^7.0.0",
-    "lerna": "^4.0.0",
+    "lerna": "^3.20.2",
```

Comment:
But we wanna keep this yo

Reply:
why tho?

Reply:
oh shit - sorry just read the diff in reverse

Reply:
one sec